doc/source/code_of_conduct.rst

```diff
@@ -0,0 +1,128 @@
+you don't feel comfortable speaking in public, etc. For these or other
+reasons, structured follow-up may be necessary and here we provide the means
+for that: we welcome reports by emailing
+`conduct@geopandas.org <mailto:conduct@geopandas.org>`__ or by filling out `this
+form <https://goo.gl/forms/CHANGETHELINK>`__.
+
+This code applies equally to founders, developers, mentors and new community
+members, in all spaces managed by the GeoPandas Project. This
+includes the mailing lists, our GitHub organization, our chat room, in-person
+events, and any other forums created by the project team. In addition,
+violations of this code outside these spaces may affect a person's ability to
+participate within them.
+
+By embracing the following principles, guidelines and actions to follow or
+avoid, you will help us make Jupyter a welcoming and productive community. Feel
+free to contact the Code of Conduct Committee at
+`conduct@geopandas.org <mailto:conduct@geopandas.org>`__ with any questions.
+
+1. **Be friendly and patient**.
+
+2. **Be welcoming**. We strive to be a community that welcomes and supports
+   people of all backgrounds and identities. This includes, but is not limited
+   to, members of any race, ethnicity, culture, national origin, color,
+   immigration status, social and economic class, educational level, sex, sexual
+   orientation, gender identity and expression, age, physical appearance, family
+   status, technological or professional choices, academic
+   discipline, religion, mental ability, and physical ability.
+
+3. **Be considerate**. Your work will be used by other people, and you in turn
+   will depend on the work of others. Any decision you take will affect users
+   and colleagues, and you should take those consequences into account when
+   making decisions. Remember that we're a world-wide community. You may be
+   communicating with someone with a different primary language or cultural
+   background.
+
+4. **Be respectful**. Not all of us will agree all the time, but disagreement is
+   no excuse for poor behavior or poor manners. We might all experience some
+   frustration now and then, but we cannot allow that frustration to turn into a
+   personal attack. It’s important to remember that a community where people
+   feel uncomfortable or threatened is not a productive one.
+
+5. **Be careful in the words that you choose**. Be kind to others. Do not insult
+   or put down other community members. Harassment and other exclusionary
+   behavior are not acceptable. This includes, but is not limited to:
+
+   -  Violent threats or violent language directed against another person
+   -  Discriminatory jokes and language
+   -  Posting sexually explicit or violent material
+   -  Posting (or threatening to post) other people's personally identifying information ("doxing")
+   -  Personal insults, especially those using racist or sexist terms
```

Comment:
I'd throw in xenophobic here to cover race & ethnicity. Is gender expression and sexuality falling under sexism?

Reply:
I'll change `racist` to `xenophobic`. I would say that gender and sexuality falls under sexism, but we can make it explicit if you feel it'd be better.

Reply:
sorry, meant `racist, sexist, and xenophobic. ` And dunno, like not sure if we should special case the common cases or if special casing leaves to "why did you leave out?"